playgrounds/geo-javascript/src/app.js

```diff
@@ -1,4 +1,4 @@
-import { instantMeiliSearch } from '../../../src/client/index'
+import { instantMeiliSearch } from '@meilisearch/instant-meilisearch'
```

Comment:
This will use the local version and not the npm one, right?

Reply:
Yes! Local version :)

Reply:
https://turbo.build/repo/docs/handbook/workspaces#workspaces-which-depend-on-each-other